README.md

```diff
@@ -74,40 +78,40 @@ This sample demonstrates how to create your Azure virtual machines with Managed
 <a id="example"></a>
 ## What does example.rb doing?
 
-This sample starts by setting up ResourceManagementClient, the resource provider clients, a resource group, and a storage account using your subscription and credentials.
+This sample starts by setting up ResourceManagementClient, the resource provider clients, a resource group, a storage account and a user assigned identity if desired, using your subscription and credentials.
 
 ```ruby
 subscription_id = ENV['AZURE_SUBSCRIPTION_ID'] || '11111111-1111-1111-1111-111111111111' # your Azure Subscription Id
-  provider = MsRestAzure::ApplicationTokenProvider.new(
-      ENV['AZURE_TENANT_ID'],
-      ENV['AZURE_CLIENT_ID'],
-      ENV['AZURE_CLIENT_SECRET'])
-  credentials = MsRest::TokenCredentials.new(provider)
-  resource_client = Azure::ARM::Resources::ResourceManagementClient.new(credentials)
-  resource_client.subscription_id = subscription_id
-  network_client = Azure::ARM::Network::NetworkManagementClient.new(credentials)
-  network_client.subscription_id = subscription_id
-  storage_client = Azure::ARM::Storage::StorageManagementClient.new(credentials)
-  storage_client.subscription_id = subscription_id
-  compute_client = Azure::ARM::Compute::ComputeManagementClient.new(credentials)
-  compute_client.subscription_id = subscription_id
+  options = {
+      tenant_id: ENV['AZURE_TENANT_ID'],
+      client_id: ENV['AZURE_CLIENT_ID'],
+      client_secret: ENV['AZURE_CLIENT_SECRET'],
+      subscription_id: subscription_id,
+  }
+
+  resource_client = Resources::Client.new(options)
+  network_client = Network::Client.new(options)
+  storage_client = Storage::Client.new(options)
+  compute_client = Compute::Client.new(options)
+  authorization_client = Authorization::Client.new(options)
 
   #
   # Managing resource groups
   #
-  resource_group_params = Azure::ARM::Resources::Models::ResourceGroup.new.tap do |rg|
-    rg.location = WEST_US
+  resource_group_params = resource_client.model_classes.resource_group.new.tap do |rg|
+    rg.location = LOCATION
```

Comment:
Where is the LOCATION defined???

Reply:
LOCATION is defined here https://github.com/veronicagg/compute-ruby-msi-vm/blob/29ece424403ac78e75c580029c21deb2cd7724c9/example.rb#L14 
I can include the definition in the readme.md file, though I didn't feel like it was something necessary to include as it doesn't need much of an explanation - constant is defined in the code in example.rb